apps/shooting-clubs/index.js

```diff
@@ -137,134 +102,56 @@ module.exports = {
-        'second-contact-address-lookup'
-      ],
-      next: '/second-contact-email',
-      locals: {
-        field: 'second-contact'
-      }
-    },
+    }),
+    '/second-contact-address-manual': secondContactAddressLookup.manual,
     '/second-contact-email': {
       fields: [
         'second-contact-email',
         'second-contact-phone'
       ],
-      next: '/location-postcode'
+      next: '/location-address'
     },
-    '/location-postcode': {
-      addressKey: 'locationAddresses',
-      template: 'postcode-loop.html',
-      controller: require('../common/controllers/postcode-loop'),
-      fields: [
-        'location-postcode'
-      ],
-      next: '/location-address',
-      backlink: 'second-contact-email',
-      forks: [{
-        target: '/location-address-lookup',
-        condition(req) {
-          const addresses = req.sessionModel.get('location-addresses');
-          return addresses && addresses.length;
-        }
-      }],
-      locals: {
-        field: 'location'
+    '/location-address': Object.assign(locationAddressLookup.start, {
+      formatAddress: (address) => address.formatted_address.split('\n').join(', ')
+    }),
+    '/location-address-select': Object.assign(locationAddressLookup.select, {
+      fieldSettings: {
+        className: 'address'
       }
-    },
-    '/location-address': {
-      addressKey: 'locationAddresses',
-      template: 'address-loop.html',
-      controller: require('../common/controllers/address-loop'),
-      fields: [
-        'location-address-manual'
-      ],
-      next: '/location-address-category',
-      prereqs: ['/location-postcode', '/second-contact-email'],
-      backlink: 'location-postcode',
-      locals: {
-        field: 'location'
-      }
-    },
-    '/location-address-lookup': {
-      addressKey: 'locationAddresses',
-      template: 'address-lookup-loop.html',
-      controller: require('../common/controllers/address-lookup-loop'),
-      fields: [
-        'location-address-lookup'
-      ],
-      next: '/location-address-category',
-      locals: {
-        field: 'location'
-      }
-    },
+    }),
+    '/location-address-manual': locationAddressLookup.manual,
     '/location-address-category': {
       template: '../common/views/add-another-address-loop.html',
-      addressKey: 'locationAddresses',
-      controller: require('./controllers/location-address-category'),
       fields: [
         'location-address-category'
       ],
       continueOnEdit: true,
-      next: '/location-add-another-address',
-      locals: {
-        field: 'location'
-      }
+      next: '/location-add-another-address'
     },
     '/location-add-another-address': {
       addressKey: 'locationAddresses',
       template: 'add-another-address-loop.html',
-      controller: require('../common/controllers/add-another-address-loop'),
-      fields: [
-        'location-add-another-address'
+      controller: require('./controllers/location-address-loop'),
```

Comment:
what controller is it if you don't have a category? is it still location-address-loop?